src/main/jhipster/src/main/webapp/app/germplasm-manager/germplasm-search.component.ts

```diff
@@ -379,6 +384,23 @@ export class GermplasmSearchComponent implements OnInit {
         });
     }
 
+    registerGermplasmUpdated() {
+        // Load all germplasm that has been updated via import.
+        this.eventSubscriber = this.eventManager.subscribe('germplasmUpdated', (event) => {
+
+            this.columnFilterComponent.clearFilters();
+
+            // Get the existing gids filter
+            const gidsFilter = this.filters.find((filter) => filter.key === 'gids');
+            gidsFilter.value = event.content.join(',');
+
+            // Manually add it to the filters and apply.
+            this.columnFilterComponent.selectedFilter = gidsFilter.key;
+            this.columnFilterComponent.AddFilter();
+            this.columnFilterComponent.updateListFilter(gidsFilter);
+        });
+    }
+
```

Comment:
This is one way to do it, though it feels a little bit constrained to the update process in my opinion. Another way would be the [`filterBy` parameter](https://github.com/IntegratedBreedingPlatform/InventoryManager/blob/a008b43d103181fc04037703aa0f71ece9c898c8/src/main/webapp/app/entities/transaction/transaction.component.ts#L426), like in transaction.component, which enables the addition of other types of programmatic filtering, while at the same it seems it doesn't require additional modifications to the filter component itself, only the `reloadFilters()`.

cc @abatac

Reply:
I tried to use the reloadFilters() before but that only refreshed the table and the GID column filter is not displayed/added. I think it works in View Transaction because the TransactionID filter is always visible.